### ISSUE TITLE
card is not synced when changes are only in attachments

### ISSUE THREAD

**Issue by mortee** [stefan-niedermann/nextcloud-deck#1221]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Describe the bug**
When I only add or remove attachments on a card in the browser, the changed don't show up in the app.

- create a card
- sync it to the app
- add an attachment in the browser interface
- sync the app
- see that the new attachment didn't show up

**Expected behavior**
The app should show the changes in only attachments.

**Versions**

 - Nextcloud: 22.2.3
 - Nextcloud Deck: 1.5.5
 - Nextcloud Android: 3.18.1
 - Nextcloud Android Deck: 1.20.0

**Smartphone (please complete the following information):**
- Device: Galaxy A52
- Android-Version: 11
- App-Store: F-Droid

**Comment by stefan-niedermann** [comment]
Hi @mortee,

thanks you for the report!

This look pretty much like [yet another issue](https://github.com/nextcloud/deck/issues?q=etag+is%3Aissue+is%3Aopen+) with not up to date `ETags` of the (not by us developed) Deck server app.

@desperateCoder can you verify this? I'll open the issue then.

@mortee if my assumption is right, the Deck Android app is not the cause of the issue, but we'll take care of reporting it at the correct location. As a workaround, you can disable temporarily `Etag` support in the Deck Android apps settings. Be aware though that each sync will take ages and create very much network traffic then.

**Comment by mortee** [comment]
@stefan-niedermann thank you! FWIW, I'm not currently affected by this issue, just noticed it, and thought it would be nice to report. I came across it while experienting with the non-appearing indicator icon for card description.

**Comment by aljawaid** [comment]
> Hi @mortee,
> 
> thanks you for the report!
> 
> This look pretty much like [yet another issue](https://github.com/nextcloud/deck/issues?q=etag+is%3Aissue+is%3Aopen+) with not up to date `ETags` of the (not by us developed) Deck server app.
> 
> @desperateCoder can you verify this? I'll open the issue then.
> 
> @mortee if my assumption is right, the Deck Android app is not the cause of the issue, but we'll take care of reporting it at the correct location. As a workaround, you can disable temporarily `Etag` support in the Deck Android apps settings. Be aware though that each sync will take ages and create very much network traffic then.

hello, I had the same issue for NC 23.x... solved when you mentioned the `Etag` option.... can't tell all my users to keep doing that though

**Comment by stefan-niedermann** [comment]
Yeah, that's why the issue is still open and I wrote "workaround" instead of "solution" 😉
